frontend/src/components/eventselect/reducer/AnalyticsEventSelect.tsx

```diff
@@ -11,84 +11,59 @@
  *  and limitations under the License.
  */
 
-import { Button, SelectProps } from '@cloudscape-design/components';
+import { Button } from '@cloudscape-design/components';
+import { OptionDefinition } from '@cloudscape-design/components/internal/components/option/interfaces';
```

Comment:
Is it safe to use interal implementation?

Reply:
Updated to `SelectProps.Option`